.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
because this is not supported by mocha, we use babel to convert the code to something that mocha understands